.github/workflows/ci.yaml

```diff
@@ -316,11 +316,11 @@ jobs:
       CARGO_INCREMENTAL: false
       # There were binding loops with layout in our demo, so ignore theses errors
       # Also ignore errors that were warnings before
-      SLINT_INTERPRETER_ERROR_WHITELIST: "is part of a binding loop;must be called;duplicated element id;Unknown type Clip;in Path"
+      SLINT_INTERPRETER_ERROR_WHITELIST: "is part of a binding loop"
```

Comment:
Right, none of these error should be there anymore, not even the binding loops, so these lines could go (also the comments)  
But it's ok to keep it there for reference if we need them later.

```suggestion
```

Reply:
Sorry for pushing another version: GH insists it can not apply your changes:-(